internal/util/operator-registry/package_manifest_test.go

```diff
@@ -17,14 +17,14 @@ package registry
 import (
 	"testing"
 
-	olmregistry "github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry"
+	"github.com/operator-framework/operator-registry/pkg/registry"
 )
 
 func TestValidatePackageManifest(t *testing.T) {
-	channels := []olmregistry.PackageChannel{
+	channels := []registry.PackageChannel{
 		{Name: "foo", CurrentCSVName: "bar"},
 	}
-	pm := &olmregistry.PackageManifest{
+	pm := &registry.PackageManifest{
```

Comment:
It shows not be the same object impl here: https://github.com/operator-framework/operator-sdk/blob/8c0bafbe2a91bf2c833dc868f9db671b9f97826e/internal/pkg/scaffold/olm-catalog/package_manifest.go#L37

Reply:
WDYT about changing the name in SDK to reflect more why it is required here and make clear that both are not the same struct/object?

Reply:
IMO, our `PackageManifest` struct name is the best name to use since it follows the same pattern as structs for other scaffolded files. It's package path makes it clear enough (to me at least) what its intended purpose is.

Reply:
Hi @joelanford,

I understand why would you prefer keeping the same name. Just to clarifies, I was just wondering if we could make it more clear in order to avoid the same mistake that this Pr has the goal of a fix happened again. 

Shows that it is very susceptible to face the same mistake, I mean another dev at any moment use one struct when would like to use another one since the SDK project is importing OLM at the end the dev will face 2 objects with the same name but with different purposes which shows be reducing the manutence ability.

Reply:
It is ok. Tks for share your inputs as well.